index.html

```diff
@@ -1178,15 +1178,18 @@ <h2>Verification Methods</h2>
                 <dd>
                   <p>
 The value of the `id` property for a [=verification method=] MUST be a
-[=string=] that conforms to the conforms to the [[URL]] syntax.
+[=string=] that conforms to the [[URL]] syntax.
                   </p>
                 </dd>
                 <dt>type</dt>
                 <dd>
-The value of the `type` property MUST be a [=string=] that references exactly
-one [=verification method=] type. In order to maximize global
-interoperability, the [=verification method=] type SHOULD be registered in
-the Data Integrity Specification Registries [TBD -- DIS-REGISTRIES].
+The value of the `type` property MUST be a [=string=] that conforms to the [[URL]] syntax,
```

Comment:
I think this needs text that includes a "mapping" -- like what the data model spec says:

> The value of the type [property](https://w3c.github.io/vc-data-model/#dfn-property) MUST be, or map to (through interpretation of the @context property), one or more [URLs](https://w3c.github.io/vc-data-model/#dfn-url).

https://w3c.github.io/vc-data-model/#types

Something like this?:

```suggestion
The value of the `type` property MUST be a [=string=] that conforms to the [[URL]] syntax
or maps to (through interpretation of the `@context` property) a [[URL]],
```

Reply:
```suggestion
The value of the `type` property MUST be a [=string=] that either conforms to the
[[URL]] syntax or maps (through interpretation of the `@context` property) to a [[URL]],
```

Reply:
```suggestion
The value of the `type` property MUST be a [=string=] that conforms to the [[URL]] syntax
or maps to (through interpretation of the `@context` property) a [[URL]],
```